src/main/scala/ai/privado/entrypoint/SyncProcessor.scala

```diff
@@ -0,0 +1,30 @@
+package ai.privado.entrypoint
+
+import ai.privado.auth.AuthenticationHandler
+import ai.privado.cache.Environment
+import ai.privado.model.Constants
+import better.files.File
+import org.slf4j.LoggerFactory
+
+object SyncProcessor extends CommandProcessor {
+  private val logger = LoggerFactory.getLogger(this.getClass)
+
+  def checkOutputFileExists(): Boolean = {
+    val repoPath       = config.sourceLocation.head
+    val outputFileName = "privado"
+    File(repoPath + "/.privado/" + outputFileName + ".json").exists
```

Comment:
@abhstabs I see we have used the output folderName and file name `.privado` and `privado` across, Let's create a  constant for them, so they are not repeated